WordPress/src/main/res/layout/stats_block_value_item.xml

```diff
@@ -38,5 +39,8 @@
         android:layout_gravity="bottom"
         android:textColor="?attr/wpColorSuccess"
         android:textSize="16sp"
-        android:visibility="gone" />
+        android:visibility="gone"
+        tools:text="-7 (-100%)"
+        android:layoutDirection="ltr"
```

Comment:
This is not a request for a change. Just curious why we set the layout direction here 🤔

Reply:
Thank you for raising the question 🙇 

I noticed this view was placing the [number sign](https://en.wikipedia.org/wiki/Sign_(mathematics)#Sign_of_a_number) inverted when using a RTL language, see screenshot:

| <img width="401" alt="non_ltr" src="https://user-images.githubusercontent.com/4588074/195109565-7b7bda24-3c57-4988-99ee-4834570a3d9f.png"> |
| - |

This isn't the case in other places because we always have a prefix / the number is part of a longer string.

Upon further investigation it appears this happens only when the text of the view contains a number and any other characters except letters.

This seems to be the behavior of text views in RTL languages, though not consistent.

A few other explorations without setting `android:layoutDirection="ltr"`:
| Prefix word | Suffix word |
| - | - |
| <img width="314" src="https://user-images.githubusercontent.com/4588074/195112078-5afbcfed-5877-43f4-9716-7c1b1ad94981.png"> | <img width="314" src="https://user-images.githubusercontent.com/4588074/195111176-a722869e-a56e-4fee-97ad-df2e2071fa0e.png"> |

Conclusion: In order to maintain a consistent position of the number sign with the other places, this change was indeed needed 🙂.